src/components/ui/Checkbox.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import * as CheckboxPrimitive from '@radix-ui/react-checkbox';
+import { Check } from 'lucide-react';
+import { forwardRef } from 'react';
+
+import { cx } from '@/lib/utils';
+
+const Checkbox = forwardRef<
+  React.ComponentRef<typeof CheckboxPrimitive.Root>,
+  React.ComponentPropsWithoutRef<typeof CheckboxPrimitive.Root>
+>(({ className, ...props }, ref) => (
+  <CheckboxPrimitive.Root
+    ref={ref}
+    className={cx(
+      'peer h-4 w-4 shrink-0 rounded-sm border border-primary ring-offset-background focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-primary data-[state=checked]:text-primary-foreground',
```

Comment:
So I have had an issue before with the checkbox from shadcn where it is round. If it is round we need to change from `rounded-sm` to `rounded-[2px]` or something like that. This is because it is in general a good practice to seperate chekboxes as being mostly square and radio buttons as round